static/lang/en.json

```diff
@@ -716,11 +716,11 @@
     "CHAT.CommandLine.Help.Pay.Title" : "Pay",
     "CHAT.CommandLine.Help.Pay.Usage.Command" : "/pay",
     "CHAT.CommandLine.Help.Pay.Usage.Example" : "<br>/pay 3gc2bp<br>/pay 10bp450ss",
-    "CHAT.CommandLine.Help.Pay.Usage.Note" : "If this command is sent by the GM, it will create instead a chat card where players are offered to pay for the amount entered by the GM.",
+    "CHAT.CommandLine.Help.Pay.Usage.Note" : "This command offers 3 options : <ul><li><strong>/pay amount</strong> : GM only. Display a chat card for all users, with a button for them to pay. Exemple : /pay 12gc2br</li><li><strong>/pay amount player_name </strong>: GM and Players. Display a chat card for the designated player only.</li><li><strong>/pay amount actor_name</strong>: GM only. Automatically reduces the money of the designated actor with the amount. Only a part of the actor name can be used : if the actor's name is Kastor Lieberung, then you can write /pay 12ss kastor.If the actor has a connected player, a chat card will ask him to approve the command. If the actor is not owned by a player (or the player is not connected), the amount is deduced automatically.</li></ul>",
     "CHAT.CommandLine.Help.Credit.Title" : "Credit",
     "CHAT.CommandLine.Help.Credit.Usage.Command" : "/credit",
-    "CHAT.CommandLine.Help.Credit.Usage.Example" : "<br>/credit 3gc2bp<br>/credit 10bp450ss each",
-    "CHAT.CommandLine.Help.Credit.Usage.Note" : "This command is GM only. An amount of money entered by the GM will be split among Players (split mode, default mode) or each of the players get the money (each mode).",
+    "CHAT.CommandLine.Help.Credit.Usage.Example" : "<br>/credit 3gc2bp<br>/credit 10bp450ss each<br>/credit 1bp420ss John<br>/credit 1gc12ss Kastor Lieberung",
+    "CHAT.CommandLine.Help.Credit.Usage.Note" : "This command is GM only. An amount of money entered by the GM will be sent to actors/players, with the following options : <ul><li><strong>/credit amount split (default)</strong> : Display as chat card for all users, with a button for them to pay. Exemple : /pay 12gc2br</li><li><strong>/pay amount player_name</strong>: GM and Players. Display a chat card for the designated player only.</li><li><strong>/pay amount actor_name </strong>: GM only. Automatically reduces the money of the designated actor with the amount. Only a part of the actor name can be used : if the actor's name is Kastor Lieberung, then you can write /credit 12ss kastor.</li></ul>",
```

Comment:
This line has some text copied & pasted from the /pay command.
Also, both Pay and Credit notes has typos ("Exemple" and "12gc2br").